### ISSUE TITLE
Port 1514 is open on harbor appliance

### ISSUE THREAD

**Issue by hmahmood** [vmware/harbor#1936]
The docker-compose files for harbor and notary have this port open for syslog. However, this service should be bound only to localhost, and not to any address (0.0.0.0).

Docker versions:
```
# docker version
Client:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.4
 Git commit:   78d1802
 Built:        Wed Jan 11 00:23:16 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.4
 Git commit:   78d1802
 Built:        Wed Jan 11 00:23:16 2017
 OS/Arch:      linux/amd64
# docker-compose version
docker-compose version 1.11.1, build 7c5d5e4
docker-py version: 2.0.2
CPython version: 2.7.13
OpenSSL version: OpenSSL 1.0.1t  3 May 2016
```

**Comment by reasonerjt** [comment]
@wy65701436 please see if we can bind it to 127.0.0.1 instead of 0.0.0.0

**Comment by reasonerjt** [comment]
fixed in #1938
